src/app.js

```diff
@@ -111,6 +112,14 @@ var app = async function(options) {
     next();
   });
 
+  // attach request-id to request object and response
+  app.use((req, res, next) => {
+    let reqId = req.headers['x-request-id'] || uuidv4();
+    req.requestId = reqId;
+    res.setHeader('x-for-request-id', reqId);
```

Comment:
Can we just echo back `x-request-id` instead of setting `x-for-request-id`?

Reply:
so my thinking was that putting a request id on a response was sort of confusing.  My thinking is that it's the response for a specific request, which is where the `-for` name came into play.

I don't have strong thoughts on either option and am happy to do either option, @petemoore what do you think?